### PR TITLE
append timer mock signatures to bindings

### PR DESCRIPTION
**Pull request by tanabarr**
Signed-off-by: Tom Nabarro <tom.nabarro@outlook.com>

### CODE REVIEW

File: fable/Bindings.fs [hunk 1]
```diff
@@ -154,6 +154,21 @@ type [<AllowNullLiteral>] JestStatic =
...
+  /// (but not any new timers that get created during that process)
+  abstract runOnlyPendingTimers: unit -> unit
+  /// All timers are advanced by msToRun milliseconds
+  abstract advanceTimersByTime: int -> unit
+  /// All timers are advanced by msToRun milliseconds
+  abstract runTimersToTime: int -> unit
+  /// Enable fake timers by calling jest.useFakeTimers();
+  /// This mocks out setTimeout and other timer functions with mock functions.
+  abstract useFakeTimers: unit -> unit
+  abstract useRealTimers: unit -> unit
```
Comment (jgrund) [inline fable/Bindings.fs]: this returns `JestStatic`, according to type definitions.

File: fable/Bindings.fs [hunk 2]
```diff
@@ -154,6 +154,21 @@ type [<AllowNullLiteral>] JestStatic =
...
+  /// Fast forward and exhaust only currently pending timers
+  /// (but not any new timers that get created during that process)
+  abstract runOnlyPendingTimers: unit -> unit
+  /// All timers are advanced by msToRun milliseconds
+  abstract advanceTimersByTime: int -> unit
+  /// All timers are advanced by msToRun milliseconds
+  abstract runTimersToTime: int -> unit
+  /// Enable fake timers by calling jest.useFakeTimers();
+  /// This mocks out setTimeout and other timer functions with mock functions.
+  abstract useFakeTimers: unit -> unit
```
Comment (jgrund) [inline fable/Bindings.fs]: this returns `JestStatic`, according to type definitions.

File: test/Bindings.test.fs [hunk 3]
```diff
@@ -3,6 +3,7 @@ module Fable.Import.Jest.Test.Bindings
 open Fable.Import.Jest.Exports
 open Fable.Import
 open Fable.PowerPack
+open Fable.Import.Jest.Matchers.Jesto
```
Comment (jgrund) [inline test/Bindings.test.fs]: Should just be able to do `open Fable.Import.Jest.Matchers`

File: test/Bindings.test.fs [hunk 4]
```diff
@@ -75,3 +76,75 @@ describe "expect" <| fun () ->
...
+      timer.Start()
+
+      jest.clearAllTimers()
+      expect.Invoke(h).toEqual(false);
+  ]
+
+  yield! testFixture withSetup [
+    "should contain runAllTimers", fun timer ->
+      let mutable h = false
+      let handler _ = h <- true
```
Comment (jgrund) [inline test/Bindings.test.fs:105]: This could be passed in the `withSetup`

Comment (tanabarr) [reply test/Bindings.test.fs:105]: tried that but it didn't work for some reason, spent a fair amount of time trying to keep as much as possible within the setup routine but ended up here

**Comment by jgrund** [comment]
Thanks!

**Pull request merged**